backend/rest_api_service/models.py

```diff
@@ -26,6 +27,19 @@ class DBCategory(Base):
 
     events = relationship("DBEvent", secondary="db_event_category", back_populates="categories")
 
+    def __init__(self, category_id, category_name, category_description):
+        self.category_id = category_id
+        self.category_name = category_name
+        self.category_description = category_description
+
+    @staticmethod
+    def create(category: str):
+        category = json.loads(category)
```

Comment:
w endpointach będzie zmiana, żeby nie używać stringów, to tutaj też trzeba będzie zmienić